src/stepfunctions/steps/states.py

```diff
@@ -254,27 +254,27 @@ def accept(self, visitor):
 
     def add_retry(self, retry):
         """
-        Add a Retry block to the tail end of the list of retriers for the state.
+        Add a Retry block or a list of Retry blocks to the tail end of the list of retriers for the state.
 
         Args:
-            retry (Retry): Retry block to add.
+            retry (Retry or list(Retry)): Retry block(s) to add.
         """
         if Field.Retry in self.allowed_fields():
-            self.retries.append(retry)
+            self.retries.extend(retry) if isinstance(retry, list) else self.retries.append(retry)
         else:
-            raise ValueError("{state_type} state does not support retry field. ".format(state_type=type(self).__name__))
+            raise ValueError(f"{type(self).__name__} state does not support retry field. ")
```

Comment:
nice! love the use of `f strings to leave the code a little better than when you found it.